go/libkb/login_state.go

```diff
@@ -187,11 +187,9 @@ func (s *LoginState) Shutdown() error {
 
 		if s.loginReqs != nil {
 			close(s.loginReqs)
-			s.loginReqs = nil // block future sends to this channel
```

Comment:
This was intended, hence the comment.  Checking for nil before sending would be racy, but sending to a close channel would cause a panic. 

Can you put the comments back?